Content.Server/Commands/Observer/Ghost.cs

```diff
@@ -14,8 +15,8 @@ public class Ghost : IConsoleCommand
         public string Command => "ghost";
         public string Description => "Give up on life and become a ghost.";
         public string Help => "ghost";
-        public bool CanReturn { get; set; } = true;
 
+        [Obsolete("Call IGameTicker.OnGhostAttempt(Mind, CanReturn) instead.")]
```

Comment:
Well you can't really `Obsolete` this since this is still used by it being a command.

Reply:
Only implicitly. I see the point, it's just... I really *really* don't want people to keep calling this directly

Reply:
Gotten rid of the attribute